input/pagecontent/medicine-information.md

```diff
@@ -18,9 +18,9 @@ The guidance below addresses how medicinal product identification can be structu
 
 ### Medicinal Product Identification
 
-For extemporaneous medications, it is expected the medication code is the primary mechanism to identify a medicine. In this case, a text only list of ingredients may be supplied or may be coded using a medicines terminology.
+For extemporaneous medications, the medication code element is the primary mechanism to identify a medicine. In this case, a list of ingredients including strength and form is recommended to be provided as text in the medication code element e.g. Medication.code.text or MedicationRequest.medicationCodeableConcept.text.
 
-For non-extemporaneous medications, the medication code (or set of codes) is the mandatory primary mechanism to identify a medicine and its defining attributes (by terminology lookup) including form and strength. 
+For non-extemporaneous medications, the medication code element is the primary mechanism to identify a medicine. In this case, information identifying the medicinal product including strength and form is recommended to be provided as a code from a terminology in the medication code element, e.g. Medication.code.coding or MedicationRequest.medicationCodeableConcept.coding, and the description of the medicinal product as presented to the user is provided in the text e.g. Medication.code.text or MedicationRequest.medicationCodeableConcept.text. 
```

Comment:
As above, apply special formatting to all instances of element names and paths.

Reply:
Applied.